app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainFragment.kt

```diff
@@ -62,8 +62,8 @@ class MainFragment : Fragment() {
 
     fun updateLocation() {
         var locationRequest = LocationRequest()
-        locationRequest.priority = LocationRequest.PRIORITY_HIGH_ACCURACY
-        locationRequest.interval = 10000
+        locationRequest.priority = LocationRequest.PRIORITY_LOW_POWER
+        locationRequest.interval = 60000
         locationRequest.fastestInterval = 5000
```

Comment:
Changed these because, as mentioned earlier, we should not need to get a high accuracy location that will just waste the battery of the user's device, and the interval shouldn't be too often as we do not need to update the location that often.

My reasoning is, the weather will probably be something that is city-wide, or at least is shown as city-wide. The interval is something that doesn't need to be updated that often, because a user will most likely be using this application when they are at home or at somewhere for a long-period of time. We don't need too much accuracy here, and we do not need to continuously update the location.

One could argue that we keep PRIORITY_HIGH_ACCURACY, but change the interval, which I would definitely be all for. We just need to make sure that we are limiting the battery usage of the user's device here with unnecessarily frequent location updates.

Reply:
This is correct, this is after the code review that I made the change to instead getting the full address, just get the city that the user is in since that what the Json Api uses ( this is update now in another branch after talking with the team). Also the user should really maybe need to push this button 1-2 time to get their location. This functionality was one, to help users  who are the board line of two city and may not know which one they fall in. 
The other was just learning how use The GPS of the phone to get the data.

Reply:
Awesome! Glad to see you went ahead and made the change. Which value did you end up using for the priority?

Reply:
right now it is the same, the change was in the data we are getting ( getting just the city not the full address).  When first making the Fun the idea was, even the button spent a lot of effort getting the location it only need to do it 1 time and it should stop.

Now if we wanted constant update to location, we would invest more time at the time of make those changes, but this should be a one and done thing thing.